### DISCUSSION TITLE
Ferramentas de no-code

### DISCUSSION THREAD

**Discussion by ifdiego** [frontendbr/forum#1754] [Perguntas e Respostas]
Olá pessoal,

Ultimamente tenho visto o surgimento de algumas ferramentas de no-code, onde você faz um site ou app inteiro sem sequer escrever uma linha de código. São os casos do [Bubble](https://bubble.io) e do [Glide](https://www.glideapps.com). Qual a opinião de vocês sobre essas soluções?

**Comment by kvnol** [comment]
Recomendo a leitura da issue #490, deu bastante pano pra manga.

**Comment by ifdiego** [comment]
Opa, valeu!

**Comment by willycamargo** [comment]
Já tive uma opinião diferente, mas atualmente acredito que essas ferramentas são excelentes opções para protótipos ou projetos simples e acho que com o passar do tempo irão aparecer novas que atendam cada vez mais necessidades específicas para projetos mais complexos.

Acho que com o avanço dessas tecnologias, muita tarefa repetitiva deixará de ser feita por mãos de profissionais especialistas (Front-end) e passarão a ser feitas diretamente por designers e "produteiros".

Entender as vantagens e limitações dessas ferramentas pode ser essencial para posições profissionais futuras na nossa área.

Só para finalizar: eu recomendaria e usaria sem problemas caso eu entendesse que a ferramenta se encaixa para a necessidade/problema apresentado.

**Comment by ifdiego** [comment]
@willycamargo Sim, eu concordo.

Quando postei a pergunta eu estava com um pé atrás, mas entre esses dias usei uma dessas ferramentas que citei, o Bubbe, em um projeto e atendeu super bem.

**Comment by kivervinicius** [comment]
Existe alguma ferramenta no-code open ou free?

**Comment by ifdiego** [comment]
@kivervinicius Totalmente eu acho que não. O Bubble que usei tem uma versão gratuita, acredito que seja o padrão dessas ferramentas.

**Comment by kivervinicius** [comment]
@ifdiego https://grapesjs.com/
